tests/cross_functional/system_test/test_mcg_replication_with_disruptions.py

```diff
@@ -404,3 +419,333 @@ def test_log_based_replication_with_disruptions(
+        s3_put_bucket_versioning(mcg_obj_session, bucket_2.name)
+        logger.info("Enabled object versioning for both the buckets")
+
+        # Enable sync versions in both buckets replication policy
+        replication_1 = json.loads(get_replication_policy(bucket_name=bucket_2.name))
+        replication_2 = json.loads(get_replication_policy(bucket_name=bucket_1.name))
+        replication_1["rules"][0]["sync_versions"] = True
+        replication_2["rules"][0]["sync_versions"] = True
+
+        update_replication_policy(bucket_2.name, replication_1)
+        update_replication_policy(bucket_1.name, replication_2)
+        logger.info(
+            "Enabled sync versions in the replication policy for both the buckets"
+        )
+
+        # Update previously uploaded object with new data and new version
+        self.upload_objects_with_retry(
+            mcg_obj_session,
+            bucket_2,
+            bucket_1,
+            mockup_logger_target,
+            test_directory_setup.origin_dir,
+            pattern=object_key,
+            prefix=prefix_2,
+        )
+        logger.info(
+            f"Updated object {object_key} with new version data in bucket {bucket_2.name}"
+        )
+
+        wait_for_object_versions_match(
+            mcg_obj_session,
+            awscli_pod_session,
+            bucket_1.name,
+            bucket_2.name,
+            obj_key=f"{prefix_2}/{object_key}",
+        )
+        logger.info(
+            f"Replication works from {bucket_2.name} to {bucket_1.name} and has all the versions of object {object_key}"
+        )
+
+        # Will perform disruptive operations and object uploads, version verifications
+        # parallely.
+        with ThreadPoolExecutor(max_workers=1) as executor:
+
+            # Update object uploaded previously from the second bucket and
+            # then shutdown the noobaa core and db pod nodes
+            noobaa_pods = [
+                get_noobaa_core_pod(),
+                get_noobaa_db_pod(),
+            ] + get_noobaa_endpoint_pods()
+            noobaa_pod_nodes = [get_pod_node(pod_obj) for pod_obj in noobaa_pods]
+            aws_cli_pod_node = get_pod_node(awscli_pod_session).name
+            for node_obj in noobaa_pod_nodes:
+                if node_obj.name != aws_cli_pod_node:
+                    node_to_shutdown = [node_obj]
+                    break
+
+            logger.info(
+                f"Updating object {object_key} with new version data in bucket {bucket_1.name}"
+            )
+            future = executor.submit(
+                self.upload_objects_with_retry,
+                mcg_obj_session,
+                bucket_1,
+                bucket_2,
+                mockup_logger_source,
+                test_directory_setup.origin_dir,
+                pattern=object_key,
+                prefix=prefix_1,
+            )
+
+            nodes.stop_nodes(node_to_shutdown)
```

Comment:
stop_nodes by default forces the instance to stop. It will ungraceful stop the instance which is not equivalent to a graceful shutdown

Reply:
yeah. but even ungraceful shutdown should be good for this.